lighthouse-core/lib/i18n/i18n.js

```diff
@@ -439,6 +439,10 @@ function _resolveIcuMessageInstanceId(icuMessageInstanceId, locale) {
   const [_, icuMessageId, icuMessageInstanceIndex] = matches;
   const icuMessageInstances = _icuMessageInstanceMap.get(icuMessageId) || [];
   const icuMessageInstance = icuMessageInstances[Number(icuMessageInstanceIndex)];
+  if (!icuMessageInstance) {
```

Comment:
this was a drive-by for fixing `-GA` which is currently broken because of messages generated during gathering that can't be replaced in auditing phase

Reply:
thank you!!

Reply:
apologies but I can't recall if it'd be fine to land this hotfix now.

Reply:
brendan's about to fix anyway i'll revert 👍